logic/src/commonMain/kotlin/com/wire/kalium/logic/data/client/ClientMapper.kt

```diff
@@ -83,6 +84,19 @@ class ClientMapper(
         )
     }
 
+    fun fromNewClientEntity(clientEntity: NewClientEntity): Client = with(clientEntity) {
+        Client(
+            id = ClientId(id),
+            type = null,
+            registrationTime = registrationDate,
+            deviceType = deviceType?.let { fromDeviceTypeEntity(deviceType) },
+            label = null,
+            model = model,
+            isVerified = true,
```

Comment:
I wouldn't expect new clients to be verified by default, but I guess doesn't really matter for what you are using this for.

Reply:
true, it's not used anywhere in this flow, but I'll change it to `false` just in case :)